proto/raft_cmdpb.proto

```diff
@@ -146,6 +146,9 @@ message SplitRequest {
     // left region use new_region_id.
     // Will be ignored in batch split, use `BatchSplitRequest::right_derive` instead.
     bool right_derive = 4 [deprecated=true];
+    // It should be false iff the region split by user key such as split table or create partion table etc, and new region's size should be zero. 
+    // It should be true iff the region's load reach the threshold such as size、keys、load check etc, and new region's size will amortize the origin region.
```

Comment:
Maybe change to "It determines whether the split-off regions amortize the region size of the source region. In some cases, such as ..., the split-off regions won't have any data, so they should't amortize the source region's region size." ?

Reply:
I will change the behavior description.